shared/src/bundles/resumes/content/education/education-dto.type.ts

```diff
@@ -0,0 +1,16 @@
+type Education = {
+    id: string;
+    resumeId: string;
+    institution: string;
+    degree: string;
+    startDate: string;
+    endDate: string;
```

Comment:
If the person is still studying, then this field should be optional

Reply:
Fixed.